test/StreamableGeneratorTest.cpp

```diff
@@ -37,8 +37,10 @@ TEST(StreamableGeneratorTest, TestEmptyGeneratorReturnsEmptyResult) {
   ASSERT_FALSE(generator.hasNext());
 }
 
+const std::string MAX_TEST_BUFFER_STRING(TEST_BUFFER_SIZE, 'A');
+
 basic_stream_generator<TEST_BUFFER_SIZE> generateMultipleElements() {
-  co_yield std::string(TEST_BUFFER_SIZE, 'A');
+  co_yield MAX_TEST_BUFFER_STRING;
   co_yield 1;
   co_yield "Abc";

```

Comment:
I was thinking about making the `"1Abc"` also a constant, but then again you yield the 1 as an integer.
Maybe the "1Abc" should still be a constant which is defined near the `generateMultipleElements`() function.

Reply:
From what I can tell it is used twice in `StreamableGeneratorTest.cpp` so I don't think it's worth introducing further abstraction here